man/fcase.Rd

```diff
@@ -36,7 +36,9 @@ data.table::fcase(
 	x < 5L, as.Date("2019-10-11"),
 	x > 5L, as.Date("2019-10-14")
 )
-# [1] "2019-10-11" "2019-10-11" "2019-10-11" "2019-10-11" NA  "2019-10-14" "2019-10-14" "2019-10-14" "2019-10-14" "2019-10-14"
+# [1] "2019-10-11" "2019-10-11" "2019-10-11"
```

Comment:
why do we need output in examples? it doesn't seems to be practical because when eventually a print/format/locale changes the print might be not-matching to the current output.

Reply:
I usually put the output commnet because they appear in the documentation and ine can see the output without having to run the code.

Reply:
I understand but I think it is much safer to not add them. Also pkgdown when render example it includes the output, so we would ended up with double output. I would ideally follow the base R convention about that.

Reply:
Fine to remove them.